test/e2e/instrumentation/logging/stackdriver/utils.go

```diff
@@ -84,8 +85,8 @@ type sdLogProvider struct {
 
 func newSdLogProvider(f *framework.Framework, scope logProviderScope) (*sdLogProvider, error) {
 	ctx := context.Background()
-	hc, err := google.DefaultClient(ctx, sd.CloudPlatformScope)
-	sdService, err := sd.New(hc)
+	hc, _ := google.DefaultClient(ctx, sd.CloudPlatformScope)
```

Comment:
We'd better deal with this error.
```suggestion
	hc, err := google.DefaultClient(ctx, sd.CloudPlatformScope)
	framework.ExpectNoError(err)
```

Reply:
thanks. done